maintainers/maintainer-list.nix

```diff
@@ -13402,4 +13402,16 @@
     github = "jpagex";
     githubId = 635768;
   };
+  portothree = {
+    name = "Gustavo Porto";
+    email = "gustavoporto@ya.ru";
+    github = "portothree";
+    githubId = 3718120;
+  };
+  antonmedv = {
+    name = "Anton Medvedev";
+    email = "anton+github@medv.io";
+    github = "antonmedv";
+    githubId = 141232;
+  };
```

Comment:
Why are you adding two maintainers? 

They should also each be added in their own commit: `maintainers: add <name>`

After that everything LGTM

Reply:
The contributing doc wasn't very clear if maintainers should be project owner/maintainer or just someone to maintain the nix module, so I have added both. 
It's means that I have to open a different PR to include maintainers or just pick the commit outside of `llama: init at 1.0.1`?

Reply:
Ah no, only the maintainer of the nix code should be added. 

A different commit is fine.

Reply:
Ok, updated